scripts/benchmarks_local.py

```diff
@@ -436,7 +436,7 @@ def generate_single_benchmark_ci_args(parsed_args: Namespace, specific_run_type:
             '--category-exclusion-filter', 'NoInterpreter', 'NoWASM', 'NoMono',
             '--wasmDataDir', os.path.join(get_run_artifact_path(parsed_args, RunType.WasmInterpreter, commit), "wasm_bundle", "wasm-data"),
             '--wasmEngine', parsed_args.wasm_engine_path,
-            '--wasmArgs', '\" --experimental-wasm-eh --expose_wasm --module\"',
```

Comment:
I guess we don't want to remove this from the local script, makes sense.

Reply:
lol!! This *is* the local script!!

Ok, so we are removing it for that so it assumes a minimum version of v8. Maybe we should check for that, or .. add somewhere that a minimum version is required?

Reply:
Do you know if there is a minimum version that we don't need the argument for? If not, I will add a note to get latest.

Reply:
@radekdoulik @pavelsavara might know the version where this was removed.

Reply:
If we find a version, we can add it in another PR. In place of that, I clarified the v8/jsvu requirement at the top.